src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.Android.cs

```diff
@@ -164,12 +164,17 @@ Action onImageSet
 				//We only need to set a background if the drawArea is non-zero
 				if (!drawArea.HasZeroArea())
 				{
-					var imageBrushBackground = background as ImageBrush;
-					if (imageBrushBackground != null)
+					if (background is ImageBrush imageBrushBackground)
 					{
 						var setBackground = DispatchSetImageBrushAsBackground(view, imageBrushBackground, drawArea, onImageSet);
 						disposables.Add(setBackground);
 					}
+					//TODO: AcrylicBrush on Android
```

Comment:
Expected ?

Reply:
Yes, Android is not yet finalized, I was told to make others mergable and finish Android in the meantime